great-idea/css/services.css

```diff
@@ -0,0 +1,118 @@
+@import url("index.css");
```

Comment:
I like your separation of concern here, separating the CSS and importing it where needed is great. 👍